sdk/documentintelligence/ai-document-intelligence-rest/test/public/analysis.spec.ts

```diff
@@ -1076,8 +1077,12 @@ describe("DocumentIntelligenceClient", () => {
         )
         .get();
 
-      // Header starts with a special character followed by "PNG"
-      assert.equal(output.body.toString().slice(1, 4), "PNG");
+      if (isUnexpected(output)) {
+        throw new Error("The response was unexpected.");
+      }
+      assert.isTrue(
+        isValidPNG(await streamToUint8Array(output.body as unknown as NodeJS.ReadableStream)),
```

Comment:
can't we eliminate this cast if we fix the model body shape here?

https://github.com/HarshaNalluru/azure-sdk-for-js/blob/29c1aea1d70621906d37d7bf2da3d963b8f5313f/sdk/documentintelligence/ai-document-intelligence-rest/src/responses.ts#L194C3-L194C7

Reply:
I can do that, but do we not want to provide `Uint8Array`s in the future?

Reply:
updated as per the discussion offline